composer.json

```diff
@@ -3,12 +3,12 @@
     "description": "Make debugging easier by adding more data to your laravel logs",
     "require": {
         "php": "^7.3|^8.0",
-        "illuminate/support": "5.6.*|5.7.*|5.8.*|^6.0|^7.0|^8.0",
+        "illuminate/support": "5.6.*|5.7.*|5.8.*|^6.0|^7.0|^8.0|^9.0",
         "monolog/monolog": "^1.23|^2.1.1"
     },
     "require-dev": {
         "phpunit/phpunit": "^9.3.9",
-        "orchestra/testbench": "^6.0.0"
+        "orchestra/testbench": "^6.0.0|^7.0.0"
     },
```

Comment:
```diff
-        "orchestra/testbench": "^6.0.0|^7.0.0"
+        "orchestra/testbench": "^7.0.0"
```

Do this and it works.

Reply:
I'm against this change. 

[v7.x of testbench requires PHP ^8.0](https://github.com/orchestral/testbench/blob/7.x/composer.json#L20), so if we make that change this package would not support PHP < 8 anymore.

@gauravmak thoughts?

Reply:
It is a `require-dev` dependency. We will need PHP 8 during development to run tests. The package can still support older versions of PHP.

Reply:
I'm pretty sure the Github workflow will fail for PHP < 8 if we removed that. Should we also update the test to drop checks for PHP < 8?

Reply:
```diff
-        "orchestra/testbench": "^6.15.1",
+        "orchestra/testbench": "^6.23|^7.0.0",
```

Reply:
`^6.15.1` is working fine and [all tests run successfully](https://github.com/eduardlleshi/laravel-log-enhancer/runs/5154265223?check_suite_focus=true) with `--prefer-lowest` flag. 

Composer will decide which version of orchestra/testbench to install. 
If `--prefer-lowest` flag is set it will install v6.0.0.
Otherwise if `--prefer-stable` is set (which is by default) it will install `v6.24.1` (whatever the latest 6.x version is). 

The change you are requesting doesn't have any impact at all, so I fail to understand the reason behind it.

I'm not being pushy, I just want to get some clarification on why we need to change it.

Reply:
As per your note on description `Side note: I tried to get PHP 8.1 support as well, but the action is failing to run the test using --prefer-lowest option. The package is working as intended on 8.1, but the tests cases are failing. I think the tests need to be re-written to support PHPUnit ^9.5.` 

I tried and it's throwing an error so I suggest you do this.

Reply:
Oh, I see. That's more complicated than that though.

PHPUnit ^9.5 is needed for PHP 8.1, but the tests are not compatible with PHPUnit ^9.5. We'd have to re-write the tests to get support for 8.1 (and be backward compatible with 7.3). I don't have the capability to handle that right now.

Reply:
there is no issue with the PHPUnit versions because as per the https://github.com/sebastianbergmann/phpunit/blob/master/composer.json#L25 they did not need more than or equal PHP 8.1.

There is an issue is with the `orchestra/testbench` because one of the dependencies of this package is not updated so we need to update this package to the `6.23` version. after this, all the tests are passed.